gradle.properties

```diff
@@ -1 +1,2 @@
 kotlin.code.style=official
+kotlin.daemon.jvmargs=-Xmx512g
```

Comment:
Without this, I got a clear error message that there's not enough memory to compile, and was hinted to add this setting. The value comes from me, though.

Reply:
g?

Reply:
Good catch, definitely m!